pkg/server/server.go

```diff
@@ -2,59 +2,58 @@
 package server
 
 import (
-	"encoding/json"
 	"fmt"
+	"github.com/cmsgov/easi-app/pkg/handlers"
+	"github.com/okta/okta-jwt-verifier-golang"
 	"log"
 	"net/http"
-	"time"
+	"os"
+	"strings"
 )
 
-type status string
-
-const (
-	draft status = "DRAFT"
-)
+func isAuthenticated(authHeader string) bool {
+	tokenParts := strings.Split(authHeader, "Bearer ")
+	bearerToken := tokenParts[1]
+	if bearerToken == "" {
+		return false
+	}
 
-type profile struct {
-	Name         string
-	SystemOwners []string
-	CreatedAt    time.Time
-	UpdatedAt    time.Time
-	Status       status
-}
+	toValidate := map[string]string{}
+	toValidate["cid"] = os.Getenv("OKTA_CLIENT_ID")
```

Comment:
This needs to be pulled out, possibly in the `cmd` directory as part of the setup. We're using cobra/pflags for the command line tool, so we'll likely want to integrate with viper to read these in (https://github.com/spf13/viper) from the CLI, then pass them as configurations to the web application portion.